docs/native-modules-ios.md

```diff
@@ -444,39 +444,41 @@ const onSubmit = async () => {
 
 ### Sending Events to JavaScript
 
-Native modules can signal events to JavaScript without being invoked directly. For example, you might want to signal to JavaScript a reminder that a calendar event from the native iOS calendar app will occur soon. The preferred way to do this is to subclass `RCTEventEmitter`, implement `supportedEvents` and call self `sendEventWithName`:
+Native modules can signal events to JavaScript without being invoked directly. For example, you might want to signal to JavaScript a reminder that a calendar event from the native iOS calendar app will occur soon. The preferred way to do this is to subclass `RCTEventEmitter`, implement `supportedEvents` and call `[self sendEventWithName:body:]`.
 
-Update your header class to import `RCTEventEmitter` and subclass `RCTEventEmitter`:
+Add subclass `CalendarManager` that inherits from `RCTEventEmitter`.
 
 ```objectivec
-//  CalendarModule.h
+//  CalendarManager.h
 
-#import <React/RCTBridgeModule.h>
 #import <React/RCTEventEmitter.h>
 
-@interface CalendarModule : RCTEventEmitter <RCTBridgeModule>
```

Comment:
What's the reason behind removing `RCTBridgeModule`?

Reply:
FYI: The Native Modules docs have been mostly rewritten not that long time ago in https://github.com/facebook/react-native-website/pull/2261, and we have fairly tested the example codes then, so I'm a bit surprised to see this change.

Reply:
Thank you for the deep dive into the logic behind the changes, I really enjoy reading that part. 🙂 

> The RCTBridgeModule protocol is not required for simple use, so I did not add it. For example, RCTKeyboardObserver and RCTStatusBarManager inherited from RCTEventEmitter in react-core do not comply with the RCTBridgeModule protocol.

The problem is that this guide should cover most common and standard use case and bridge removal might seems link nice optimization, but lack of it will break most of the Native Modules anyway and probably some of the website users following the tutorial will report issues with missing bridge in their cases.

So it would be nice to retain the Bridge here, other thank that the changes looks good, great work! 👍 

Also please fix the lint warnings, without that I cannot merge this in.